pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
```

Comment:
> @smarterclayton or anyone else want to chime in? I don't have a strong
> opinion - hard to think of any name that won't be at least mildly
> misleading to somebody. But I only want to change this one more time this
> week :)

I request something closer.  A `ClusterResourceOverrideConfig` that doesn't mutate a `Cluster`-scoped thing or even general `Resource` is pretty hard to defend.